src/App.tsx

```diff
@@ -15,71 +15,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component {
+  state: Readonly<State> = {
+    selectedGood: 'Jam is',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleClearButton = () => {
+    this.setState({ selectedGood: 'No goods' });
+  };
```

Comment:
'No goods' - isn’t selected good too, it's message 
you can set the selected good to null or empty string and check if null, show 'No goods', if not, show message with good